### ISSUE TITLE
Entity support for executing batch sql

### ISSUE THREAD

**Issue by shin-nakamura** [future-architect/uroborosql#102]
I want to execute batch sql using entity, but it isn't exists. 
Please add entity support for executing batch sql.

```Java
// For insert
int insertBatch(List<?> entityList);

// For update
int updateBatch(List<?> entityList);
```

**Comment by HidekiSugimoto189** [comment]
Hi @shin-nakamura .
This Issue was implemented by @ota-meshi .
It becomes available in v0.10.0.
Please try using it.

**Comment by HidekiSugimoto189** [comment]
Closed this issue.
